numerapi/numerapi.py

```diff
@@ -1,12 +1,13 @@
 # -*- coding: utf-8 -*-
 
+from __future__ import absolute_import
+
 # System
 import zipfile
 import os
 import logging
 import datetime
 import decimal
-from typing import List, Dict
```

Comment:
I added type hints on purpose, any reason for removing them?

Reply:
not intended